packages/auth/src/grant/routes.ts

```diff
@@ -413,6 +414,17 @@ async function continueGrant(
     accessToken
   })
 }
+async function deleteGrant(
```

Comment:
This should also check that the `Authorization` header contains the correct continue access token. [It's in the form of](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-5.4) `GNAP $accessToken`, and `$accessToken` should be compared against a grant's `continueToken` field.

For future reference when https://github.com/interledger/rafiki/issues/882 gets worked on, grant deletion should also fall under the scope of this issue

Reply:
Shouldn't this be done by a middleware?

Reply:
Yeah :point_down: would move the token checks to a middleware
- #882